scripts/runner.go

```diff
@@ -46,3 +46,5 @@ func GetRepoRootDir() (string, error) {
 	dirPath := filepath.Dir(filepath.Dir(filename))
 	return dirPath, nil
 }
+
+
```

Comment:
are those a result of changes required by the go source checks?

Reply:
I'm trying to verify that the go checks in this PR are actually working in the ci, these lines will be removed once I'm sure the checks work